schemas/json/layout/layout.schema.v1.json

```diff
@@ -117,6 +117,9 @@
           ]
         }
       },
+      "pageBreak": {
+        "$ref": "#/definitions/pageBreak"
+      },
```

Comment:
It is for all components, so that you can e.g. have a new page before a heading which is a GenericComponent.
Specifically it is implemented in SummaryComponent, GenericComponent, and DisplayGroupContainer

Reply:
Right, because you could co-mingle any component on the print page if you wanted.. I see!